From a81dbcc6cad2bbd1c7f4e6e7f67b66ee072f6b88 Mon Sep 17 00:00:00 2001
From: Evgeni Golov <evgeni@debian.org>
Date: Thu, 22 Jun 2017 08:22:07 +0200
Subject: [PATCH 10/11] add cgroupfs-mount to Should-Start/Stop sysvinit LSB
 headers

otherwise init might try to start lxcfs before cgroupfs was mounted,
which will result in an empty /var/lib/lxcfs/cgroup and weird issues.

Debian-Bug: https://bugs.debian.org/859219
Signed-off-by: Evgeni Golov <evgeni@debian.org>
---
 config/init/sysvinit/lxcfs | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/init/sysvinit/lxcfs b/config/init/sysvinit/lxcfs
index 21a9c61..bdbcb34 100755
--- a/config/init/sysvinit/lxcfs
+++ b/config/init/sysvinit/lxcfs
@@ -6,6 +6,8 @@
 # Provides:             lxcfs
 # Required-Start:       $remote_fs
 # Required-Stop:        $remote_fs
+# Should-Start:         cgroupfs-mount
+# Should-Stop:          cgroupfs-mount
 # Default-Start:        2 3 4 5
 # Default-Stop:         0 1 6
 ### END INIT INFO
-- 
2.13.2

